tests/foreman/api/test_discoveryrule.py

```diff
@@ -87,8 +87,6 @@ def test_positive_create_with_org_loc(self):
         :id: 121e0a30-8a24-47d7-974d-998886ed1ea7
 
         :expectedresults: Rule was created and with given org & location.
-
-        :CaseImportance: Critical
```

Comment:
Since this is functional test on Acceptance level, it should be moved to tier3 (tier1 is reserved for importance: Critical, level: Component and type: Functional).

I think similar comment might apply to many other tests here, but GitHub doesn't make it easy for me to see that.

If you want to clean up tiers in another PR, I'm OK with that.

Reply:
Updated Metadata as per your Doc.